raiden/network/proxies/token_network.py

```diff
@@ -169,7 +169,7 @@ def new_netting_channel(
             )
             raise RuntimeError('creating new channel failed')
 
-        channel_identifier = self.detail_channel(partner)['channel_identifier']
+        channel_identifier = self.detail_channel(self.node_address, partner)['channel_identifier']
```

Comment:
note: this interface doesn't support opening a channel for another address, which was one of the intended usaged (that's why we require both addresses, instead of using the transaction sender)

Reply:
I'm aware of that, but this is currently not possible in the `ChannelManager` either and I'm not sure when we would do that in the Raiden client.